src/main/java/mate/controller/CreateManufacturerController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+@WebServlet("/manufacturers/create")
+public class CreateManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/manufacturer/add.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String country = req.getParameter("country");
+        String name = req.getParameter("name");
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setCountry(country);
+        manufacturer.setName(name);
+        manufacturerService.create(manufacturer);
+        req.getRequestDispatcher("/WEB-INF/views/manufacturer/add.jsp")
+                .forward(req, resp);
+    }
```

Comment:
did you check this endpoint?
I think it doesn't work and you can't create a new manufacturer